.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
I'd give a bit more time to react to the stale-bot, maybe 2 weeks instead of one. But that's just a gut feeling